### PR TITLE
Add test for Put Objects legalhold status end point in Console API

### PR DESCRIPTION
**Pull request by cniackz**
Fixes https://github.com/miniohq/engineering/issues/385

Add test for `Put Object's legalhold status` end point in Console API

### CODE REVIEW

**Comment by prakashsvmx** [comment]
@cniackz Please resolve conflicts.

**Review by bexsoft — APPROVED** [review]

File: integration/buckets_test.go [hunk 1]
```diff
@@ -2357,6 +2381,104 @@ func TestDeleteObjectsRetentionStatus(t *testing.T) {
...
+			// 3. Put Objects Legal Status
+			putResponse, putError := PutObjectsLegalholdStatus(
+				bucketName,
+				prefix,
+				status,
+				tt.args.versionID,
+			)
+			if putError != nil {
+				log.Println(putError)
+				assert.Fail("Error creating the bucket")
```
Comment (kaankabalak) [inline integration/buckets_test.go:2469]: Should we change the error message here, as from what I have understood, we're not creating a bucket here?

Comment (cniackz) [reply integration/buckets_test.go:2469]: Good point, I will resolve conflict and change the error message too.
Thanks!

**Review by adfost — CHANGES_REQUESTED** [review]
Fix merge conflicts.

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**